### DISCUSSION TITLE
Video Preview Thumbnails?

### DISCUSSION THREAD

**Discussion by SimonDaBoss** [croneter/PlexKodiConnect#1459] [Q&A] [answered]
I am using PlexKodiConnect on kodi matrix on my desktop, when I play a video everything is fine but when I'm on the original plex client, I skip the video but also can see thumbnails to see where I'm skipping to. Is this not available in plexkodiconnect?

**Comment by croneter** [comment, marked as answer]
No, that is not available

**Reply by SimonDaBoss** [reply]
Thanks for the answer, do you think it'll ever be implemented or is this a Kodi limitation?

**Comment by cipioh** [comment]
I was just searching for this feature and found this question/response.  Now that it's been nearly two years... is this something than can be implemented at all? Just migrating to Kodi/PKC and that's really the only native Plex feature that I miss.

**Reply by SimonDaBoss** [reply]
Won't happen sorry, always kept me away from Kodi. Sticked to plex since.
